backend/EasySpeak.Core/EasySpeak.Core.Common/DTO/Lesson/LessonWebDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace EasySpeak.Core.Common.DTO.Lesson;
+
+public class LessonWebDto
+{
+    public long Id { get; set; }
+    public string? Name { get; set; }
+    public UserDto? User { get; set; }
+    public string? Description { get; set; }
+    public string? MediaPath { get; set; }
+    public DateTime StartAt { get; set; }
+    public LanguageLevelDto LanguageLevel { get; set; }
+    public int Subscribers { get; set; }
```

Comment:
More meaningful names, Suscribers - list of users, it can't be int

Reply:
As we decided on meeting the only number of Subscribers is needed, that is why I used int. I think it will be better to rename property to SubscrubersCount for understanding it's meaning. What do you think?

Reply:
renamed